package.json

```diff
@@ -6,7 +6,7 @@
     "scripts": {
         "dev": "next dev -p 3002",
         "build": "next build",
-        "start": "next start",
+        "start": "next start -p 3002",
```

Comment:
I added this as I wasn't sure if we wanted this here for production builds? or does the port not really matter when deployed?

Reply:
It will matter as we need to know what to include in the IaC for security group rules and environment variable endpoint values for SA and research containers. I think the options are:

* Expose production at the same port as dev via `3002`
* Modify the `Dockerfile` to use the `3000` port for production in the healthcheck, and override the dev health check `docker-compose.yml`. (I think this is what @sparksam was thinking)

I have no strong preference personally :shrug: .

Reply:
Something like this for dev envs? 
```
services:
    trusted-output:
        env_file: .env
        container_name: trusted-output
        environment:
            CI: $CI
        build:
            context: .
            dockerfile: Dockerfile
        volumes:
            - ./src:/app/src
        restart: always
        networks:
            - trusted-output
        ports:
            - 3002:3002
        # For local dev only, assuming devs are just running `docker compose up`
        command: ['npm', 'run', 'dev']
        healthcheck:
            test: curl --fail http://localhost:3002/health || exit 1
            interval: 30s
            timeout: 30s
            retries: 3
            start_period: 20s
          
        # For sad linux users
        extra_hosts:
            - 'host.docker.internal:host-gateway'
```

Reply:
I think this is resolved by @philschatz 's work on our call just now